src/scripts/admin/filters/image-enhanced.js

```diff
@@ -13,6 +13,13 @@ registerBlockStyle('core/image', {
 	label: 'Drop shadow',
 });
 
+registerBlockStyle('core/image', {
+	name: 'square',
+	label: 'Variable',
```

Comment:
is `Variable` the right word, variable in what?  I'm not sure i understand this image block style, and its purpose

Reply:
There is a description that goes along with the name which outlines the variable nature of the style.

Reply:
yes i saw, this, not sure that the label variable still defines this. Maybe something like
`Image ratio sizes` , `variable ratios`, `screen variable ratios`. i see your difficulty here with the name... just thing `variable` is too vague

Reply:
I don't disagree, however the real estate is limited and typically relies on the popup to add context, however 'Variable ratios' might just fit... I'll update it with the next commit.
<img width="606" alt="image" src="https://user-images.githubusercontent.com/5200460/208548682-d5504854-5d4b-4f43-91ff-b7cdeba41852.png">